### PR TITLE
Update haute-ecole-de-gestion-de-geneve-iso-690.csl

### PR DESCRIPTION
**Pull request by grolimur**
removed "archive" field from bibliography

### CODE REVIEW

File: haute-ecole-de-gestion-de-geneve-iso-690.csl [hunk 1]
```diff
@@ -356,15 +415,14 @@
               <text macro="title" suffix=". "/>
             </if>
           </choose>
-          <text variable="version" prefix="version " suffix=". "/>
+          <text variable="version" prefix="Version " suffix=". "/>
```
Comment (adam3smith) [inline haute-ecole-de-gestion-de-geneve-iso-690.csl:418]: `version` now is a term you can use. Also, do you still need this now that `software` is available as an item type & mapped from Zotero?

**Review by adam3smith — COMMENTED** [review]
One quick question above

File: haute-ecole-de-gestion-de-geneve-iso-690.csl (hunk 1 above)
Comment (grolimur) [reply haute-ecole-de-gestion-de-geneve-iso-690.csl:418]: With `software` available as a doctype in CSL, this line is not useful anymore. I will remove it.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**